BigQuery.Linq.Tests/Builder/StandardQueries.cs

```diff
@@ -505,13 +505,6 @@ public void EnumAsValue()
         [TestMethod]
         public void DateTime()
         {
-            var jpCalendar = CultureInfo.GetCultureInfo("ja-jp").Calendar; // +9
-            new BigQueryContext().Select(() => new { dt = new DateTime(2014, 10, 17, 6, 0, 0, jpCalendar) })
-                .ToString()
-                .Is(@"
-SELECT
-  '2014-10-16 21:00:00.000000' AS [dt]".TrimSmart());
-
```

Comment:
It was removed because it fails when the current culture is `en-GB`. The `DateTime` constructor that takes in a calendar instance uses the calendar instance to determine if the provided values are supported (2014 is a valid year, 10 is a valid month, etc).

If you wanted to check timezone difference's being correctly applied, a better test would be:
```cs
var timeSpan = TimeSpan.FromHours(9);
            new BigQueryContext().Select(() => new { dt = new DateTimeOffset(2014, 10, 17, 6, 0, 0, timeSpan) })
                            .ToString()
                            .Is(@"
SELECT
  '2014-10-16 21:00:00.000000' AS [dt]".TrimSmart());
```

However, that feels a bit like testing the `DateTimeOffset` implementation.